### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by kousani0416**

### CODE REVIEW

File: aiagents/crew/crew.py [hunk 1]
```diff
@@ -30,6 +31,25 @@ def StartCrew(configuration: Initialize):
 
     tasks = Tasks(configuration=configuration, agents=agent_dict)
 
+    embedding = {
```
Comment (pranav-bhatt) [inline aiagents/crew/crew.py:34]: does this work? i didnt know inline if else was allowed in python

File: aiagents/panel_utils/panel_stylesheets.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
+alert_stylesheet = """
```
Comment (pranav-bhatt) [inline aiagents/panel_utils/panel_stylesheets.py:1]: why was this created?

File: aiagents/panel_utils/panel_utils.py [hunk 3]
```diff
@@ -80,15 +81,15 @@ def on_chain_start(
     def on_chain_end(self, outputs: dict[str, Any], *args, **kwargs):
         print(dumps(outputs, indent=2))
         print(self.agent_name)
-        if "This output contains the appropriate swagger metadata file to use for the task at hand" in outputs["output"]:
+        if "this output contains the appropriate swagger metadata file to use for the task at hand" in outputs["output"].lower():
```
Comment (pranav-bhatt) [inline aiagents/panel_utils/panel_utils.py:84]: can you explain the purpose of this?

**Review by pranav-bhatt — APPROVED** [review]

File: aiagents/crew/crew.py (hunk 1 above)
Comment (kousani0416) [reply aiagents/crew/crew.py:34]: yes, it is allowed in Python. This part works.

File: aiagents/panel_utils/panel_stylesheets.py (hunk 2 above)
Comment (kousani0416) [reply aiagents/panel_utils/panel_stylesheets.py:1]: To edit and customize Panel Layouts and Components, such that the UI looks better

File: aiagents/panel_utils/panel_utils.py (hunk 3 above)
Comment (kousani0416) [reply aiagents/panel_utils/panel_utils.py:84]: This is needed for multiple swagger API support and is where we figure out which swagger metadata file was selected for the task, that file name is then used as a key to find the corresponding API endpoint and bearer token.

**Pull request merged**